documents/README.md

```diff
@@ -146,22 +182,21 @@ Triple hat references allow for inline rendering or repeated calls to the same r
 They use the same attribute syntax as literate code blocks and recognise the following keys:
 
 - `v` (or `visualize`) indicates what symbols and expressions to render.
-  In the most common scenario the value is a single Elm constant, e.g. `v=spec1`.
-  Unlike in code blocks this attribute cannot be used as a tag, i.e. without `=`.
+  In the most common scenario the value is a single Elm constant of type [Spec](https://package.elm-lang.org/packages/gicentre/elm-vegalite/latest/VegaLite#Spec), e.g. `v=myChart`.
+
+* `r` (or `raw`) indicates what symbols and expressions to output without formatting.
```

Comment:
We should manually replace `*` with `-`. Because there were two empty lines between list items once, they were parsed and independent lists and got kept as such. Now we have several lists here, each containing one item.

Reply:
Fixed.